sdk/schemaregistry/azure-data-schemaregistry-apacheavro/src/main/java/com/azure/data/schemaregistry/apacheavro/SchemaRegistryApacheAvroSerializer.java

```diff
@@ -90,9 +90,9 @@
      *     encoding the object.
      * @throws NullPointerException if the {@code object} is null or {@code typeReference} is null.
      */
-    public <T extends MessageWithMetadata> T encodeMessageData(Object object, TypeReference<T> typeReference,
+    public <T extends MessageWithMetadata> T serializeMessageData(Object object, TypeReference<T> typeReference,
```

Comment:
Update the javadoc too to change from "Encodes an object into a message." to "Serializes an object into a message.". Update other JavaDocs of other APIs too.